### ISSUE TITLE
Unable to retrieve username! Most likely: incorrect login or password!

### ISSUE THREAD

**Issue by yukinarija** [jaroslawhartman/withings-garmin-v2#11]
Hello,

Is this still working as is?
Thanks.

-------

withings-garmin-v2 <..>$ ./sync.py 
Can't read config file config/withings_user.json
***************************************
*         W A R N I N G               *
***************************************

User interaction needed to get Authentification Code from Withings!

Open the following URL in your web browser and copy back the token. You will have *30 seconds* before the token expires. HURRY UP!
(This is one-time activity)

https://account.withings.com/oauth2_user/authorize2?response_type=code&client_id=183e03e1f363110b3551f96765c98c10e8f1aa647a37067a1cb64bbbaf491626&state=OK&scope=user.metrics&redirect_uri=https://wieloryb.uk.to/withings/withings.html&

Token : 14e0bd27d3f81b8a4266e1ac8f23c113fff74b23
Withings: Get Access Token
Withings: Refresh Access Token
Withings: Get Measurements
   Measurements received
'NoneType' object has no attribute 'group'
Unable to retrieve username! Most likely: incorrect login or password!
Bad response during GC upload: 403

**Comment by jaroslawhartman** [comment]
Yes; I think you're missing Garmin user and password.

It should be in form
```
/sync.py --garmin-username=<garmin user> --garmin-password=<password>
```

Or hardcode it in `sync.py`

Best regards,
Jarek

**Comment by jaroslawhartman** [comment]
Fixed in Commit 79e5e55d737a5f794365e5bb991051e92cec3b4f

Added a check if both Garmin user and password is set:

```
jhartman@wieloryb:~/scripts/withings-garmin-v2$ ./sync.py
Garmin username or password not set!
```
